src/main/java/uk/gov/hmcts/sscs/domain/BenefitType.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.sscs.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.Value;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Value
+@Builder
```

Comment:
if you provide a public constructor then don't think you use the Builder at all. You might be putting this annotation when in reality you are not using it at all.